planning/behavior_velocity_planner/src/utilization/util.cpp

```diff
@@ -23,6 +23,39 @@ namespace behavior_velocity_planner
 {
 namespace planning_utils
 {
+SearchRangeIndex getPathIndexRangeWithLaneId(
```

Comment:
No need to determine collision by simply returning the search range index indicated by lane id

Reply:
fixed at [b2ec900](https://github.com/autowarefoundation/autoware.universe/pull/405/commits/b2ec900dc819f407eb4bd9b254f3aec87563108f)
```
SearchRangeIndex getPathIndexRangeIncludeLaneId(
  const autoware_auto_planning_msgs::msg::PathWithLaneId & path, const int64_t lane_id);
```